### ISSUE TITLE
Rearrange Time Settings possibilities

### ISSUE THREAD

**Issue by LeoColman** [LeoColman/Petals#267]
We currently don't cover many of the standard time values. Furthermore, the default value should be set to HH:mm

**Comment by Anshu-Bijarnia** [comment]
Can you please elaborate this issue? i am new to open source contributions and would love to contribute.

**Comment by LeoColman** [comment]
Absolutely!
My starting idea came from not being able to choose a 12h format in the time settings list.

https://unicode-org.github.io/cldr-staging/charts/latest/by_type/date_&_time.generic.html#4f4895c28b605986

We have a lot of different time formats, and the app doesn't support many of them. My idea was to find what are the main ones and sort them in the list https://github.com/LeoColman/Petals/blob/8acbd1beab38eea8b43bf2c3caa698c232e04884/app/src/main/kotlin/br/com/colman/petals/settings/SettingsRepository.kt#L24

**Comment by LeoColman** [comment]
I started a bit overengineering with that unicode thing, so it can be simpler 😅

**Comment by Anshu-Bijarnia** [comment]
Dont you think just a simple HH:mm is enough for the 24 hours format and a simple HH:mm am/pm for 12 hours format. That would be easier for the users to choose from too. 
Just 2 options in the Time format - 12-hour/ 24-hour format

**Comment by Anshu-Bijarnia** [comment]
> Probably, yes.
> I know there are other formats used around the world such as HH.mm, but we don't need to include too many of them.
> 
> I'm ok with changing to 2 main options + having the options to display seconds or not
> 
> - HH:mm
> - hh:mm a
> - HH:mm:ss
> - hh:mm:ss a

By a here you mean am/pm right?

**Comment by LeoColman** [comment]
yes

**Comment by Anshu-Bijarnia** [comment]
br.com.colman.petals.settings.SettingsRepositoryTest > Defaults time format to HH-mm FAILED
I think changing this to HH:mm will make this test case pass.
Should i do this change in the Unit test or you should be the one doing that? I am new to open source.

**Comment by LeoColman** [comment]
Thanks a lot for the pull request @Anshu-Bijarnia !

>  I am new to open source.

You're doing perfectly fine. Even inspired me to focus on the project a bit today 😄 
If you need some more resources, https://github.com/freeCodeCamp/how-to-contribute-to-open-source this repository is very resourceful.

> Should i do this change in the Unit test or you should be the one doing that?

There's no rule about that. If you have the capacity (i.e. enough time), the expertise and the motivation, go ahead and make the change to the unit test! It will be part of the code review process. If you are more on the side "Hey, I didn't see the test and this was all I could contribute" then we can manage it in a different way.

Bottom line: Yes, do whatever it takes for your PR to pass all the checks 😄

**Comment by Anshu-Bijarnia** [comment]
Okay i am going to make the change in the unit test, and try to see if it passes all the checks. Just trying to learn more everyday.
